packages/react-admin-date-picker/src/DateRangePicker.sc.ts

```diff
@@ -0,0 +1,62 @@
+import { styled } from "@vivid-planet/react-admin-mui";
+/* tslint:disable */
+import "react-dates/initialize";
+import "react-dates/lib/css/_datepicker.css";
+/* tslint:enable */
+
+export const DateRangePickerWrapper = styled.div`
+    * {
+        font: inherit;
+    }
+
+    .DateRangePickerInput_clearDates__small {
+        display: flex;
+    }
+
+    .DateRangePickerInput_clearDates_default:hover {
+        background: unset;
+    }
+
+    .DayPickerNavigation_button__default {
+        border: none;
+    }
+
+    .CalendarDay__selected_span {
+        background: #7d98b0;
+        color: white;
+    }
+
+    .CalendarDay__selected_span:hover {
+        background: #607384;
```

Comment:
Those accent colors are probably different for each application and should be configureable via props.